### PR TITLE
[Clean up] Use yapf to lint files; Remove dangling file links

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Review by beets — APPROVED** [review]
interesting that default yapf with google style uses tabstop=4, not 2

**Comment by pradh** [comment]
> interesting that default yapf with google style uses tabstop=4, not 2

I've found our data repo setting at 4 slightly painful, but thought it was intentional.  I'd be all for switching to 2.

File: cloudbuild.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ steps:
     entrypoint: /bin/sh
     args:
       - -c
-      - "pip3 install -r requirements.txt && python3 -m pytest"
```
Comment (pradh) [inline cloudbuild.yaml:7]: Didn't realize this repo used pytest.  Is it in some way better than unittest?

**Review by pradh — APPROVED** [review]

File: cloudbuild.yaml (hunk 1 above)
Comment (beets) [reply cloudbuild.yaml:7]: iirc from 2 years ago, it's better at handling module imports and discovery (we could say - find and run all files with *test.py from this dir). in the data repo though, we'd run into issues with how we're using FLAGS

File: datacommons/examples/places.py [hunk 2]
```diff
@@ -24,50 +24,60 @@
...
+    print('Get place stats -- 2014')
+    stats = dc.get_stats(['geoId/05', 'geoId/06', 'dc/madDcid'],
+                         'dc/0hyp6tkn18vcb',
+                         obs_dates=['2014'])
+    print(stats)
 
-  print('Get place stats -- 2014 badly formatted')
-  stats = dc.get_stats(['geoId/05', 'geoId/06', 'dc/madDcid'], 'dc/0hyp6tkn18vcb', obs_dates='2014')
-  print(stats)
+    print('Get place stats -- 2014 badly formatted')
```
Comment (juliawu) [inline datacommons/examples/places.py:61]: Suggestion: specify how format is wrong. Perhaps "2014, badly formatted obs_dates"?

File: datacommons/places.py [hunk 3]
```diff
@@ -26,225 +26,243 @@
...
+                if not time_series:
+                    continue
+                for date in list(time_series):
+                    if date not in obs_dates:
+                        time_series.pop(date)
+                res[geo] = stats
+    return res
+
+
+def get_related_places(dcids,
```
Comment (juliawu) [inline datacommons/places.py:197]: This function is really cool!

**Review by juliawu — APPROVED** [review]
Focused my review on docstrings, LGTM!

File: cloudbuild.yaml (hunk 1 above)
Comment (pradh) [reply cloudbuild.yaml:7]: thanks.  both those are pretty painful today.  medium term wonder if it's worth switching data repo to pytest, or something better...

**Comment by shifucun** [comment]
Updated indent to 2

File: cloudbuild.yaml (hunk 1 above)
Comment (shifucun) [reply cloudbuild.yaml:7]: Checking online, looks like pytest is more flexible, simple than unittest. It has many plugins which might be useful for the complicated case in data repo.

**Pull request merged**